db/migrate/20210113213459_create_rdfs_class_nodes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateRdfsClassNodes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :rdfs_class_nodes do |t|
+      t.string :uri
```

Comment:
This column needs a unique index.

Reply:
Reviewing this and previous comments @excelsior